WorkloadManagementSystem/JobWrapper/JobWrapper.py

```diff
@@ -273,6 +273,12 @@ def execute( self, arguments ):
     os.environ['DIRACSITE'] = DIRAC.siteName()
     self.log.verbose( 'DIRACSITE = %s' % ( DIRAC.siteName() ) )
 
+    os.environ['DIRAC_CORES'] = str( self.ceArgs.get( 'Cores', 1 ) )
+    self.log.verbose( 'DIRAC_CORES = %s' % ( self.ceArgs.get( 'Cores', 1 ) ) )
+
+    os.environ['DIRAC_WHOLENODE'] = str( self.ceArgs.get( 'WholeNode', False ) )
+    self.log.verbose( 'DIRAC_WHOLENODE = %s' % ( self.ceArgs.get( 'WholeNode', False ) ) )
+
```

Comment:
They are not used yet but could be used by the multicore job payload itself in the same way than DIRACJOBID. It would be necessary for a job to know how many cores it has to use (especially in the wholenode case).